frontend/modelarchive/src/main/java/org/pytorch/serve/archive/Manifest.java

```diff
@@ -1,44 +1,26 @@
 package org.pytorch.serve.archive;
 
 import com.google.gson.annotations.SerializedName;
-import java.util.LinkedHashMap;
-import java.util.Map;
 
 public class Manifest {
 
-    private String specificationVersion;
-    private String implementationVersion;
+    private String createdOn;
     private String description;
```

Comment:
@harshbafna How can user's specify description for the .mar files? At present there is no documented CLI option for this: https://github.com/pytorch/serve/blob/master/model-archiver/README.md#arguments

Reply:
Cleaned up unused params.